install/build.py

```diff
@@ -9,6 +9,7 @@
 
 minimum_cuda_version = 6050
 minimum_cudnn_version = 2000
+minimum_cusolver_cuda_version = 8000
```

Comment:
This PR only supports CUDA 8 because cusolver in CUDA 7 provides only a limited range of features (e.g., SVD does not support full rank, helper functions in QR decomposition are not provided). I think it is a reasonable choice to support CUDA 8 only (it would be better to support CUDA 7 in the future, though)